api/resolvers/sub.js

```diff
@@ -166,10 +166,59 @@ export default {
           OFFSET $4
           LIMIT $5`, ...range, user.id, decodedCursor.offset, limit)
 
+      if (me) {
+        const userSubNames = await models.subSubscription.findMany({
+          where: { userId: me.id },
+          select: { subName: true }
+        })
+        const subscribedSet = new Set(userSubNames.map(s => s.subName))
+        const mutedSubNames = await models.muteSub.findMany({
+          where: { userId: me.id },
+          select: { subName: true }
+        })
+        const mutedSet = new Set(mutedSubNames.map(s => s.subName))
+        subs.forEach(sub => {
+          sub.meSubscription = subscribedSet.has(sub.name)
+          sub.meMuteSub = mutedSet.has(sub.name)
+        })
+      } else {
+        subs.forEach(sub => {
+          sub.meSubscription = false
+          sub.meMuteSub = false
+        })
+      }
+
```

Comment:
this is not the way we do this normally. when someone is logged in, this resolver now requires 3 roundtrips to the database. normally, we add `meWhatever` fetching to the main query.